### PR TITLE
Add hyperlink in README.md and add Dinner menu in Menu.md

### PR DESCRIPTION
**Pull request by Siraphop4Nene**
please merge my Dinner Menu to Master

### CODE REVIEW

**Comment by sittipat-chandang** [comment]
Good work!

**Review by sirateek — APPROVED** [review]
LGTM 👍

**Review by wave49192 — COMMENTED** [review]
Nice work!

**Review by sittipat-chandang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,3 +9,6 @@
 | Name      | Role      | Github          |
 |:----------|-----------|-----------------|
 | Siraphop Maepeasart | Dinner menu | @Siraphop4Nene |
+| :-------- | --------- | --------------- |
+| Your name | Your menu | @your_github_id |
```
Comment (sirateek) [inline README.md]: Please remove these lines

**Review by sirateek — CHANGES_REQUESTED** [review]

**Review by sirateek — APPROVED** [review]
LGTM. 👍

**Review by NamoSmith — APPROVED** [review]
Good!

**Review by wave49192 — APPROVED** [review]

**Pull request merged**